tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,94 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: upload a file with virus to a user share
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads file "filesForUpload/filesWithVirus/<filename>" to "/Shares/uploadFolder/<newfilename>" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                   |
+      | Virus found in <newfilename>. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/Shares/uploadFolder/<newfilename>" should not exist
+    And as "Alice" file "/uploadFolder/<newfilename>" should not exist
+    Examples:
+      | dav-path-version | filename      | newfilename |
+      | old              | eicar.com     | file1.txt   |
+      | old              | eicar_com.zip | file2.zip   |
+      | old              | eicarcom2.zip | file3.zip   |
+      | new              | eicar.com     | file1.txt   |
+      | new              | eicar_com.zip | file2.zip   |
+      | new              | eicarcom2.zip | file3.zip   |
+
+
+  Scenario Outline: upload a file with virus to a user share using spaces dav endpoint
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads a file "filesForUpload/filesWithVirus/<filename>" to "/uploadFolder/<newfilename>" in space "Shares" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                   |
+      | Virus found in <newfilename>. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/Shares/uploadFolder/<newfilename>" should not exist
+    And as "Alice" file "/uploadFolder/<newfilename>" should not exist
+    Examples:
+      | filename      | newfilename |
+      | eicar.com     | file1.txt   |
+      | eicar_com.zip | file2.zip   |
+      | eicarcom2.zip | file3.zip   |
```

Comment:
I think we can only test with one zip file. Also let's name the files like this
```suggestion
      | eicar_com.zip | virusFile2.zip  | 
```

Reply:
ok